PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py

```diff
@@ -131,7 +76,10 @@ def applySubstructure( process, postfix="" ) :
         e.toModify(getattr(process,'nb1AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
         e.toModify(getattr(process,'nb2AK8PuppiSoftDrop'+postfix), cuts = ['pt > 999999', 'pt > 999999', 'pt > 999999'] )
 
-
+    from RecoJets.JetProducers.nJettinessAdder_cfi import Njettiness
+    addToProcessAndTask('NjettinessAK8Subjets'+postfix, Njettiness.clone(), process, task)
+    getattr(process,"NjettinessAK8Subjets"+postfix).src = cms.InputTag("ak8PFJetsPuppiSoftDrop"+postfix, "SubJets")
+    getattr(process,"NjettinessAK8Subjets").cone = cms.double(0.8)
```

Comment:
this should be available with just ```process.NjettinessAK8Subjets.cone = ``` .. that's unless a ```+postfix``` is missing here

please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
Here it will be ```process.NjettinessAK8Subjets.cone = 0.8``` and ```getattr(process,"NjettinessAK8Subjets"+postfix).src = "ak8PFJetsPuppiSoftDrop"+postfix+":SubJets"```

Reply:
https://github.com/cms-sw/cmssw/pull/28077/files#r331037650 needs to be addressed

Reply:
please confirm that here on L82 there should be no postfix.
if the plain ```NjettinessAK8Subjets``` should be here, I'm curious where it is created.